wgpu/src/lib.rs

```diff
@@ -3150,7 +3151,16 @@ impl<'a> RenderPass<'a> {
 
     /// Draws primitives from the active vertex buffer(s).
     ///
-    /// The active vertex buffers can be set with [`RenderPass::set_vertex_buffer`].
+    /// The active vertex buffer(s) can be set with [`RenderPass::set_vertex_buffer`].
+    /// Does not use an Index Buffer. If you need this see [`RenderPass::draw_indexed`]
+    ///
+    /// Panics if vertices Range is not within 0..BufferLen / BufferStride.
+    ///
+    /// vertices: The range of vertex's being used from the Vertex Buffer.
```

Comment:
need to reformulate to account for not using vertex buffer and using many vertex buffers

Reply:
not sure what or how for this?

Reply:
new version works for me :)